src/gui/userstatus.h

```diff
@@ -42,23 +42,16 @@ class UserStatus : public QObject
     QUrl icon() const;
 
 private slots:
-    void slotFetchUserStatusFinished(const QJsonDocument &json, const int statusCode);
+    void slotFetchUserStatusFinished(const QJsonDocument &json, int statusCode);
 
 signals:
     void fetchUserStatusFinished();
 
 private:
     Status stringToEnum(const QString &status) const;
-    
-    // it needs to match the Status enum
-    const QHash<QString, int> _preDefinedStatus{{"online", 0},
-                                               {"dnd", 1}, //DoNotDisturb
-                                               {"away", 2},
-                                               {"offline", 3},
-                                               {"invisible", 4}};
-
+    QString enumToUserString(Status status) const;
```

Comment:
@camilasan shouldn't it be named just `enumToString` without **user**? Otherwise, the opposite conversion method should also contain **user** and should become `stringToEnum` -> `userStringToEnum`. Otherwise, I can notice inconsistency in naming.

Reply:
Kevin used this name as suggestion in the previous PR...